api-specificatie/openapi.yaml

```diff
@@ -1026,12 +1028,7 @@ components:
               type: array
           type: object
         _links:
-          properties:
-            adresseerbareObjecten:
-              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalPaginationLinks'
-            self:
-              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
-          type: object
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalPaginationLinks'
```

Comment:
Volgens mij was de vorige specificatie goed. In de nieuwe versie ontbreekt de self link en die moet er bij Hal altijd zijn.

Reply:
In de component waarnaar gerefereerd wordt zit een self-link

Reply:
De self-link zit in de component waarnaar gerefereerd word.